index.html

```diff
@@ -0,0 +1,231 @@
+        located in the heart of Santiago, Chile. We offer a variety of programs
+        to help you achieve your culinary goals, whether you’re looking to
+        become a professional chef, start a new career, or just expand your
+        cooking skills.
+      </p>
+      <div class="main-time">
+        <h3 class="main-sub-heading">Registration starts 2022.10.15(MON)</h3>
+        <a class="zoom" href="#">Register now for Zoom link</a>
+      </div>
+    </section>
+
+    <section id="program" class="middle-section">
+      <h2 class="program">The Program</h2>
+
+      <div class="middle-section-container">
+        <div class="middle-section-container2-1">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">Professional Chef Program</h3>
+          <p class="middle-section-para">
+            The Professional Chef Program is a 12-month, full-time program that
+            will teach you the culinary skills and business knowledge you need
+            to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-2">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Pastry Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Pastry Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+        <div class="middle-section-container2-3">
+          <img class="middle-section-img" src="chef.png" alt="chef" />
+          <h3 class="middle-section-sub-heading">
+            Professional Culinary Program
+          </h3>
+          <p class="middle-section-para">
+            The Professional Culinary Program is a 12-month, full-time program
+            that will teach you the culinary skills and business knowledge you
+            need to succeed in the culinary industry.
+          </p>
+        </div>
+      </div>
+
+      <button type="button" class="btn">See the full program</button>
+    </section>
+    <div class="teacher-topic">
+      <h2 class="teacher">Featured Teachers</h2>
+    </div>
+    <div class="container"></div>
+
+    <section class="partner-section">
+      <h2 class="partner">Our Partners</h2>
+      <div class="line"></div>
+      <ul class="partner-elments">
+        <li>
+          <img class="partner-img" src="google.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="airbnb.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="apple.png" alt="partner1" />
+        </li>
+        <li>
+          <img class="partner-img" src="youtube.png" alt="partner1" />
+        </li>
+      </ul>
+    </section>
```

Comment:
- [ ] [OPTIONAL] If you pay attention to the `horizontal part` of the elements in the `partner section`, there is a lot of distance between the elements, and make the `color` and `size` of the partner `images` like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)!

![image](https://user-images.githubusercontent.com/63915024/191834285-3e534ae1-0918-40d3-bb95-27e5c47b5256.png)